src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue2923.cs

```diff
@@ -13,18 +13,11 @@ public Issue2923(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "First tab does not load until navigating";
 
-	// [Test]
-	// public void Issue2923TestOne()
-	// {
-	// 	App.WaitForElement(q => q.Marked("FirstPageLabel"));
-	// 	App.Screenshot("First Tab is showing");
-	// }
-
-	// [Test]
-	// public void Issue2923TestTwo()
-	// {
-	// 	App.Tap(q => q.Marked("ResetButton"));
-	// 	App.Screenshot("Tabs Reset");
-	// 	App.WaitForElement(q => q.Marked("ResetPageLabel"));
-	// }
+	[Test]
+	public void Issue2923TestOne()
+	{
+		App.WaitForElement("FirstPageLabel");
+		App.Tap("ResetButton");
+		App.WaitForElement("ResetPageLabel");
```

Comment:
Can include `App.Screenshot("Tabs Reset");`
The method exists in the Appium extension methods.

Reply:
@jsuarezruiz , In this test case, we primarily focused on validating the view presented in the UI. Therefore, we used `WaitForElement` to ensure that the view is present. According to our understanding, `App.Screenshot` only captures an image and does not perform any validation. If we still need to include screenshot validations, could we use `VerifyScreenshot` instead of `App.Screenshot`?

Reply:
Yeah, that's correct, `App.Screenshot` only takes a screenshot bot does not perform any comparison with a reference one etc.